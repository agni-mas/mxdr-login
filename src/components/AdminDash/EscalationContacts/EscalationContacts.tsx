import react, { FC } from 'react';
import GridItem from '../../layout/GridItem/GridItem';
import EscalationContactsTable from '../../global/table/Tables/EscalationContactsTable';


const styles = {
    
}

interface EscalationContactsProps {}

const EscalationContacts: FC<EscalationContactsProps> = () => (
    <GridItem title="Escalation Contacts">
        <EscalationContactsTable />
    </GridItem>
);

export default EscalationContacts;