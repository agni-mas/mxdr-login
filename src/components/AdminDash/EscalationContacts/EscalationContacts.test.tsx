import React from 'react';
import { render } from "@testing-library/react";
import EscalationContacts from './EscalationContacts';


it("renders <EscalationContacts />", () => {
    const { asFragment } = render(<EscalationContacts /> );
    expect(asFragment()).toMatchSnapshot();
});
