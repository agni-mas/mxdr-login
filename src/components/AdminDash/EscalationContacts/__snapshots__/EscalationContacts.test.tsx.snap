// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <EscalationContacts /> 1`] = `
<DocumentFragment>
  <section
    class="item"
  >
    <h2
      class="grid-item-title"
    >
      Escalation Contacts
    </h2>
    <div
      class="MuiTableContainer-root css-rorn0c-MuiTableContainer-root"
    >
      <table
        aria-label="incidentId table"
        class="MuiTable-root border table css-rqglhn-MuiTable-root"
      >
        <thead
          class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
        >
          <tr
            class="MuiTableRow-root MuiTableRow-head css-ubb257-MuiTableRow-root"
          >
            <th
              class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
              scope="col"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                data-testid="GroupIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16 11c1.66 0 2.99-1.34 2.99-3S17.66 5 16 5c-1.66 0-3 1.34-3 3s1.34 3 3 3zm-8 0c1.66 0 2.99-1.34 2.99-3S9.66 5 8 5C6.34 5 5 6.34 5 8s1.34 3 3 3zm0 2c-2.33 0-7 1.17-7 3.5V19h14v-2.5c0-2.33-4.67-3.5-7-3.5zm8 0c-.29 0-.62.02-.97.05 1.16.84 1.97 1.97 1.97 3.45V19h6v-2.5c0-2.33-4.67-3.5-7-3.5z"
                />
              </svg>
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-yanhwr-MuiTableCell-root"
              scope="col"
            >
              Contacts
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-yanhwr-MuiTableCell-root"
              scope="col"
            >
              Role
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-hsoe6c-MuiTableCell-root"
              scope="col"
            >
              Location
            </th>
            <th
              class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-rmn4b8-MuiTableCell-root"
              scope="col"
            >
              Connect
            </th>
          </tr>
        </thead>
        <tbody
          class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
        />
      </table>
    </div>
  </section>
</DocumentFragment>
`;
